components/visuals/maps/AlumniOrigin.tsx

```diff
@@ -54,7 +56,12 @@ const AlumniOrigin: FC<Props> = ({
 
   const { data: filteredApplicants, isLoading: filteredApplicantsIsLoading } =
     useSWRImmutable<Awaited<ReturnType<typeof getCountryWithApplicantCount>>>(
-      "/api/data/country/count/applicant?level=" + level,
+      department
+        ? "/api/data/country/count/applicant?level=" +
+            level +
+            "&department=" +
+            department
+        : "/api/data/country/count/applicant?level=" + level,
```

Comment:
I blieve this can be simplified to
```suggestion
`/api/data/country/count/applicant?level=${level}&department=${department}`,
```

Reply:
Hi Jakob, i have run this commit and it's fine for the department page, but it will affect "http://localhost:3000/applicants/origin-country" page. In this case, the origin-country page will not select any records in any levels, i guess due to the argument.  That's why i wrote the codes in this way: if we needn't to select for the department, then we just need to specify the level for the AlumniOrigin component, which didn't has the 'department' parameter.

Reply:
Thanks for pointing this out @Pei-Nie! 🙏